packages/wallets/src/features/cfd/modals/MT5ErrorModal/MT5ErrorModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { TSocketError } from '@deriv/api-v2/types';
+import { useTranslations } from '@deriv-com/translations';
+import { WalletError } from '../../../../components';
+
+type TProps = {
+    error: TSocketError<'mt5_new_account'>['error'];
+    onClickHandler: () => void;
+};
+
+type TGetErrorConfigValues = {
+    error: TSocketError<'mt5_new_account'>['error'];
+    history: ReturnType<typeof useHistory>;
+    localize: ReturnType<typeof useTranslations>['localize'];
+};
+
+const getErrorConfig = ({ error, history, localize }: TGetErrorConfigValues) => {
+    switch (error.code) {
+        case 'ASK_FIX_DETAILS':
+            return {
+                buttonText: localize('Update profile'),
+                description: localize('To continue, provide the required information in the Personal details section.'),
+                //@ts-expect-error need to proper type history
```

Comment:
@george-usynin-deriv not sure if we have set standards for expected errors but as I can see in p2p they are adding `-` we can also use `:` but it's up to the convention we want to follow and implement in deriv-app. 

```// @ts-expect-error: need to properly type 'history'```

Reply:
@prince-deriv let me update the comment